library/src/main/java/org/xmtp/android/library/Group.kt

```diff
@@ -213,49 +214,49 @@ class Group(
         return libXMTPGroup.isActive()
     }
 
-    fun addedByInboxId(): String {
-        return libXMTPGroup.addedByInboxId()
+    fun addedByInboxId(): InboxId {
+        return InboxId(libXMTPGroup.addedByInboxId())
     }
 
     fun permissionPolicySet(): PermissionPolicySet {
         return PermissionPolicySet.fromFfiPermissionPolicySet(permissions.policySet())
     }
 
-    suspend fun creatorInboxId(): String {
-        return metadata().creatorInboxId()
+    suspend fun creatorInboxId(): InboxId {
+        return InboxId(metadata().creatorInboxId())
     }
 
     suspend fun isCreator(): Boolean {
-        return metadata().creatorInboxId() == client.inboxId
+        return metadata().creatorInboxId() == client.inboxId.value
     }
 
-    suspend fun addMembers(addresses: List<String>) {
```

Comment:
Big breaking change here is `addMembers` now takes a `inboxId` instead of a address... but because they are both strings it will blow up only after it's been hit in the code.